pyzo/core/editor.py

```diff
@@ -259,6 +259,7 @@ def __init__(self, parent, **kwds):
         self.setWrap(bool(pyzo.config.view.wrap))
         self.setHighlightCurrentLine(pyzo.config.view.highlightCurrentLine)
         self.setLongLineIndicatorPosition(pyzo.config.view.edgeColumn)
+        self.setHighlightMatchingBracket(pyzo.config.view.highlightMatchingBracket)
```

Comment:
Ones that is done, maybe it also makes sense to add an option to the view menu to enable/disable.